comps/llms/text-generation/vllm/docker/Dockerfile.microservice

```diff
@@ -1,9 +1,10 @@
-
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
 FROM langchain/langchain:latest
 
+ARG ARCH="cpu"  # Set this to "cpu" or "gpu"
```

Comment:
Currently we don't support GPU, but it's OK to add this option for later extension.

Reply:
I'd leave it.
As far as I know, some `pip` users don't like the `--extra-index-url` option and they can still build this with `gpu` if they want.